src/bin/runners/github-action-runner.ts

```diff
@@ -6,13 +6,18 @@ import { EntryPoint } from "@bc/domain/entry-point";
 import Container from "typedi";
 import { JobSummaryService } from "@bc/service/job-summary/job-summary-service";
 import { defaultFlowResult } from "@bc/domain/flow";
+import { ActionArguments } from "@bc/service/arguments/action/action-arguments";
 
 export class GithubActionRunner extends Runner {
   constructor() {
     super(EntryPoint.GITHUB_EVENT);
   }
 
   async execute(): Promise<void> {
+    // parse arguments
+    const args = Container.get(ActionArguments);
```

Comment:
sorry but I don't get why this is needed? :thinking:

Reply:
To read the input from the workflow file. Without this, the inputs don't actually get parsed (the e2e tests actually helped me figure this out :) )